release-19.2: build: touch lib/*.so files

otan opened the pull request:
TeamCity can't resolve empty directories as artifacts, so add dummy
files when doing mkrelease.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52172)
<!-- Reviewable:end -->

otan committed:
release-19.2: build: touch lib/*.so files

TeamCity can't resolve empty directories as artifacts, so add dummy
files when doing mkrelease.

Release note: None

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/52172#-:-MDpfXk49KjdGn_xo2kq:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->
